org.eclipse.jdt.debug.jdi.tests/pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright (c) 2012, 2022 Eclipse Foundation and others.
+  All rights reserved. This program and the accompanying materials
+  are made available under the terms of the Eclipse Distribution License v1.0
+  which accompanies this distribution, and is available at
+  http://www.eclipse.org/org/documents/edl-v10.php
+  This is an implementation of an early-draft specification developed under the Java
+  Community Process (JCP) and is made available for testing and evaluation purposes
+  only. The code is not compatible with any specification of the JCP.
+  
+ 
+  Contributors:
+     IBM Corporation - initial implementation
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>eclipse.jdt.debug</artifactId>
+    <groupId>eclipse.jdt.debug</groupId>
+    <version>4.25.0-SNAPSHOT</version>
+  </parent>
+  <groupId>org.eclipse.jdt</groupId>
+  <artifactId>org.eclipse.jdt.debug.jdi.tests</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <packaging>eclipse-test-plugin</packaging>
```

Comment:
> 15:16:16  [ERROR] Internal error: org.eclipse.tycho.core.osgitools.OsgiManifestParserException: Exception parsing OSGi MANIFEST /home/jenkins/agent/workspace/eclipse.jdt.debug-github_PR-93/org.eclipse.jdt.debug.jdi.tests/META-INF/MANIFEST.MF: Manifest file not found -> [Help 1]
15:16:16  org.apache.maven.InternalErrorException: Internal error: org.eclipse.tycho.core.osgitools.OsgiManifestParserException: Exception parsing OSGi MANIFEST /home/jenkins/agent/workspace/eclipse.jdt.debug-github_PR-93/org.eclipse.jdt.debug.jdi.tests/META-INF/MANIFEST.MF: Manifest file not found

CI reports an error about missing MANIFEST.MF in jdi.tests.

Currently jdi.tests is just a simple eclipse project and not an OSGI plugin, tycho cannot run it. We need to convert it to a plugin as well.

Reply:
I tried to add a MANIFEST.MF to jdi.tests and then run AutomatedSuite with JUnit Plug-in Test, it failed. Currently the AbstractJDITest launcher works as a `localJavaApplication`, but not `JunitLaunchConfig`. Looks like we have to refactor the AbstractJDITest launcher to support running it as a JUnit Plug-in Test. Any expert familar with this area and have any thoughts? @SarikaSinha

Reply:
> We need to convert it to a plugin as well.

Or run it with maven, not tycho?

Reply:
If we run it with maven instead of tycho, one problem we have to solve is that `org.eclipse.jdt.debug.jdi.tests` project depends on the JDI artifact of another project `org.eclipse.jdt.debug`.

I did a quick try and added jdt.debug into the dependencies part of the org.eclipse.jdt.debug.jdi.tests pom.xml, but looks like running tests in CI cannot recognize the dependency org.eclipse.jdt.debug.

```xml
<dependency>
      <groupId>org.eclipse.jdt</groupId>
      <artifactId>org.eclipse.jdt.debug</artifactId>
      <version>3.20.0-SNAPSHOT</version>
    </dependency>
```